torch/csrc/distributed/rpc/py_rref.cpp

```diff
@@ -285,22 +286,56 @@ c10::IValue PyRRef::toIValue() const {
   return IValue(rrefPtr);
 }
 
-void PyRRef::backward(int64_t dist_autograd_ctx_id, bool retain_graph) {
-  if (rref_->isOwner()) {
-    const auto& value =
-        c10::static_intrusive_pointer_cast<const OwnerRRef>(rref_)->getValue();
+void PyRRef::backward(int64_t autogradContextId, bool retainGraph) {
+  backward(autogradContextId, retainGraph, rref_);
+}
+
+void PyRRef::backward(
```

Comment:
curious, what's the reason for adding this static method to PyRRef  instead of adding an instance method backward to RRef?

Reply:
I didn't want to add a Python dependency to RRef.